src/ocl/convolutionocl.cpp

```diff
@@ -630,7 +630,7 @@ static void DirConvFindCore(Handle& handle,
                 conv.group_count > 1 ? callGemmStridedBatched : callGemm,
                 (conv.group_count > 1 || wDesc.GetType() == miopenInt8 ||
                  wDesc.GetType() == miopenInt8x4 || wDesc.GetType() == miopenBFloat16)
-                    ? GemmBackend_t::rocblas
+                    ? GemmBackend_t::miopentensile
```

Comment:
What if miopentensile is not enabled?

Reply:
https://github.com/ROCmSoftwarePlatform/MIOpen/pull/139/files#r479479017

Reply:
I see. If so, then `GemmBackend_t::miopentensile` looks confusing. Shali we rename it to `GemmBackend_t::miopentensile_or_rocblas` and provide a comment saying that when/if miopentensile is not available, GEMM uses rocblas?

Reply:
Maybe not now. In fact we use the same rule for rocblas and miopengemm.

Reply:
Okay. Please do that when you have time/will.